rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -59,5 +57,10 @@ class Quote {
     errorNode.innerText = this.errors.join(", ");
   }
 
+  handleSuccess(event) {
+    $("#submitQuoteModal").modal("hide");
+    $("#successModal").on("hide.bs.modal", () => { location.reload() })
```

Comment:
This is adding a listener, should have stayed in `addListeners` (perhaps changing it's callback to a `this.handleSuccessClose`). What I meant about `handleSuccess` is that you should be displaying `successModal` in here, no? otherwise what shows it?
```javascript
handleSuccess(event) {
  $("#submitQuoteModal").modal("hide");
  $("#submitQuoteModal").modal("show");
}
```

Reply:
fixed